guide/faq/extend-mission-portal-help-menu.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Extend Mission portal help menu
```

Comment:
I think I would put this together with the content about protected content. Both of them are related to "Extending Mission Portal". I would put them in the same page with dfferent headings for Adding Protected Content and Adding Custom user menu entries.

Reply:
Yeah, good idea, honestly I wasn't sure about the place of these documentations.